app.rb

```diff
@@ -79,8 +79,8 @@ def get_logger(progname, threshold=nil)
 use Rack::Locale
 
 helpers do
-  def t(*args)
-    I18n.t(*args)
+  def t(*args, **kwargs)
+    I18n.t(*args, **kwargs)
```

Comment:
Ruby generally uses more keyword arguments, which have had major changes in Ruby 3. Without this, keyword arguments are no longer passed and translations break when paramaterised translations are used (in one place).